chcemvediet/templates/main/homepage/about.html

```diff
@@ -24,25 +24,37 @@
   <div class="col-md-8 col-sm-6">
     {% include "main/homepage/casestudy/main.html" %}
   </div>
-  <div class="chv-hidden-md col-sm-12"></div>
+</div>
   {% if user.is_authenticated %}
     {# FIXME: Show something to authenticated user; See #110 #}
   {% else %}
-    <div class="col-md-4 col-sm-6 hidden-xs">
-      {% include "main/homepage/texts/why_register.html" %}
-      <div class="text-center chv-above-1">
-        <a class="btn chv-btn-wide chv-btn-inv-blue" href="{% url 'account_signup' %}">
-          {% trans 'main:homepage:signup:button' %}
-        </a>
+    <div class="row">
+      <div class="col-md-4 col-sm-6 hidden-xs">
+        {% include "main/homepage/texts/why_register.html" %}
+        <div class="text-center chv-above-1">
+          <a class="btn chv-btn-wide chv-btn-inv-blue" href="{% url 'account_signup' %}">
```

Comment:
@martinmacko47 Signup button je o 2px nizssie ako Login button. Je to problem? Moze to ostat?

Reply:
To je koli tomu, ze vyska captcha box nie je nasobkom baseline. Ak sa da fixnut vyska captcha box, tak ju pls fixni. Ak nie, tak pred/zan pridaj margin/padding, co to vykompenzuje.

Reply:
Pouzil som natvrdo hodnotu 1px, pretoze nemam nikde v konstante vyssku recaptcha, takze si to neviem dopocitat.

Reply:
Nebolo by lepsie proste:
```
<div class="g-recaptcha chv-height-4" ...>
```
A mas to presne. Nemusis riesit, ci je rozdiel 1 alebo 2px.

Reply:
:ok: dakujem